### PR TITLE
Quick equip prioritizes putting objects into belts

### LINKED ISSUE #10556: equip shorcut should put tools inside toolbelts
because reasons

### PR DESCRIPTION
**Pull request by JustSumBody**
Closes #10556 
Quick equip will now prioritize placing an item in an equipped belt rather than in the suit storage or pockets. Functionality for other items to have this property is in place through quick_store and can_quick_store. 
Now with 90% less hardcode
I never realized how much I really wanted this until I went from borg to carbon and kept dropping things.

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -578,9 +578,15 @@ var/list/slot_equipment_priority = list( \
 //puts the item "W" into an appropriate slot in a human's inventory
 //returns 0 if it cannot, 1 if successful
 /mob/proc/equip_to_appropriate_slot(obj/item/W)
-	if(!istype(W)) return 0
+	if(!istype(W))
+		return 0
 
 	for(var/slot in slot_equipment_priority)
+		if(get_item_by_slot(slot))
```
Comment (ComicIronic) [inline code/modules/mob/mob.dm]: You shouldn't really have to call this twice.

```
var/obj/item/S = get_item_by_slot
if(S)
```

Comment (JustSumBody) [inline code/modules/mob/mob.dm]: oh true

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -578,9 +578,13 @@ var/list/slot_equipment_priority = list( \
...
 //returns 0 if it cannot, 1 if successful
 /mob/proc/equip_to_appropriate_slot(obj/item/W)
-	if(!istype(W)) return 0
+	if(!istype(W))
+		return 0
 
 	for(var/slot in slot_equipment_priority)
+		var/obj/item/S = get_item_by_slot(slot)
+		if(S && S.can_quick_store(W))
+			return (S.quick_store(W))
```
Comment (PJB3005) [inline code/modules/mob/mob.dm]: The outer parentheses shouldn't be needed.

Comment (Intigracy) [inline code/modules/mob/mob.dm]: Parkinson's law of triviality is C. Northcote Parkinson's 1957 argument that members of an organisation give disproportionate weight to trivial issues. He observed that a committee whose job was to approve the plans for a nuclear power plant spent the majority of its time on discussions about relatively minor but easy-to-grasp issues, such as what materials to use for the staff bike-shed, while neglecting the proposed design of the plant itself, which is far more important but also a far more difficult and complex task.

**Comment by ComicIronic** [comment]
You know, let's just see if this causes any issues.

**Pull request merged**